umap/templates/umap/search_bar.html

```diff
@@ -3,7 +3,11 @@
     <div class="row">
       <form action="{% url 'search' %}" method="GET">
       <div class="col two-third mwide">
-        <input name="q" type="search" placeholder="{% trans 'Search maps' %}" value="{{ q|default:"" }}" />
+        {% if q %}
```

Comment:
I am OK this is bad to duplicate code :+1: 
Is there a way to write this template to prevent VariableDoesNotExist exception?
I read Django documentation and post on StackOverflow, but I don't find any way to do so.

May be, we can not "fix" this.